### PR TITLE
feat: better logging and fix deployment

### PR DESCRIPTION
**Pull request by zifeo**
MET-115

### CODE REVIEW

**Review by afmika — APPROVED** [review]

File: typegate/tests/utils.ts [hunk 1]
```diff
@@ -488,11 +491,13 @@ export async function recreateMigrations(engine: Engine) {
   const migrationsBaseDir = join(thisDir, "prisma-migrations");
 
   for await (const runtime of runtimes) {
+    console.log(runtime);
     const prisma = new PrismaMigrate(engine, runtime, null);
     const { migrations } = await prisma.create({
       name: "init",
       apply: true,
     } as any);
+    console.log("aaa");
```
Comment (Natoandro) [inline typegate/tests/utils.ts:500]: Is this supposed to be here?

File: whiz.yaml [hunk 2]
```diff
@@ -118,7 +118,7 @@ website_docgen:
         npx docusaurus docs:generate:graphql:prisma-migration
         npx docusaurus docs:generate:changelog
     depends_on:
-        - typegate1
```
Comment (Natoandro) [inline whiz.yaml:121]: Does not this depend on `typegate`?

**Review by Natoandro — APPROVED** [review]

**Review by Natoandro — DISMISSED** [review]

File: whiz.yaml (hunk 2 above)
Comment (zifeo) [reply whiz.yaml:121]: Typegate never finishes. I always thought that Whiz should have a way to manage those kind of task : e.g. if no stdout is detected after a while and the task is marked a "continuous".

Comment (Natoandro) [reply whiz.yaml:121]: You are right. Something we could implement is to check when there is a server listening on some port.

**Review by Natoandro — APPROVED** [review]

**Pull request merged**